docs/modules/ROOT/pages/how-to-guides/proc_release_plan.adoc

```diff
@@ -0,0 +1,61 @@
+= Creating a `releasePlan` object
+
+The development team creates a ReleasePlan` object in the developer workspace. The `ReleasePlan` object includes a reference to the application that the development team wants to release, along with the namespace and workspace where the application is suppose to release.
+
+.*Prerequisites*
+
+* You have an existing Development workspace.
+* Ensure you have installed `oc`.
+* You have completed the steps listed in the link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/getting-started/getting_started_in_cli/[Getting started in the CLI] page.
+
+.*Procedures*
+
+. Create a `releasePlan.yaml` object locally.
+
++
+*Example `release_plan.yaml` object*
+
++
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan <.>
+metadata:
+ labels:
+   release.appstudio.openshift.io/auto-release: 'true'
+ name: sre-production <.>
+ namespace: dev-workspace <.>
+spec:
+ applications:
```

Comment:
The changes from the last diff I reviewed are pretty big... (it was 2 files now it is 5) so it is hard for me to compare and tell whether or not the things I commented on are fixed. Nevertheless, I commented that application should be applications for the ReleasePlanAdmission, not the ReleasePlan. I see the change was made in the ReleasePlan. This is not right

Reply:
@johnbieren I created two pages, but then I realized there was already some content around `ReleasePlanAdmission`, which was hidden and published, and **my content would have been just a duplicate of the original content.** - https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/proc_managed_services_onboarding/

I don't think, I'm going to create new files.

I'm sorry for making the change in ReleasePlan, I may have jumped the gun, but I thought that since we have `application` in `ReleasePan`, I should make it applications there as well. 

I'll correct it. Let me know if you have questions.